src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -354,6 +358,10 @@ export const CheckoutPayment = ({
     return <CheckoutErrorModal isVisible onRetry={onRetry} />
   }
 
+  if (isPageLoading) {
+    return <LoadingPage loading />
+  }
```

Comment:
Is it a problem that we unmount the rest of the components on the page? Should this rather be a modal on-top of the rest of the content?

Reply:
Okey cool!